tests/jballerina-unit-test/src/test/resources/test-src/enums/enums.bal

```diff
@@ -115,13 +115,24 @@ function testEnumInRecursiveArray() {
     Keyword u = "union";
     assert(u, "union");
 
-    ApproxType b = "'boolean";
+    ApproxType b = "boolean";
+    assert(b, "boolean");
     assert(b, 'boolean);
```

Comment:
Missed this previously, shall we also add a separate test like `testEnumMemberContainingQuotedIdentifier`? Although this test also tests it, it's actually intended to test a different scenario (enum in recursive array).